doc/build-unix.md

```diff
@@ -142,6 +142,26 @@ libqrencode (optional) can be installed with:
 
     sudo dnf install qrencode-devel
 
+Dependency Build Instructions: Arch Linux
+-------------------------------------
+Build requirements:
+
+sudo pacman -S gcc-c++ libtool make autoconf automake openssl-devel libevent-devel boost-devel libdb4-devel libdb4-cxx-devel zeromq binutils m4 make pkg-config miniupnpc colorcore
+
+opcional libs:
```

Comment:
only it:

    boost
    boost-libs
    zeromq (libxs, zeromq-git)
    autoconf (make)
    automake (automake-1.11, automake-1.13, automake-1.14, automake-1.7) (make)
    binutils (binutils-git, binutils-hjl-git, binutils-tune-bfd-hash) (make)
    libtool (make)
    m4 (make)
    make (make3) (make)
    pkg-config (pkgconf-git, pkgconf) (make)
    miniupnpc (optional) – build with support for UPnP

Not optinal?? 
font: https://aur.archlinux.org/packages/bitcoin-core/